dvc/data_cloud.py

```diff
@@ -75,7 +76,7 @@ def get_remote(self, remote=None, command="<command>"):
         )
 
     def _init_remote(self, remote):
-        config = self.repo.config.get_remote_settings(remote)
+        config = get_remote_settings(self.repo.config, remote)
```

Comment:
same as with pkg - why don't we just pass the name to the Remote object and let it take care of extracting setting from wherever it needs to extract them from?